### PR TITLE
[doc] Fix outdated classname in example

### PR DESCRIPTION
**Pull request by jryan128**
The class is in a new place.

## Describe the PR

Updates outdated documentation.

### CODE REVIEW

**Comment by jsotuyod** [comment]
Thanks!

**Comment by pmd-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="true">No regression tested rules have been changed.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: docs/pages/pmd/userdocs/extending/your_first_rule.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ copy-paste into your ruleset XML. The resulting element looks like so:
 <rule name="DontCallBossShort"
       language="java"
       message="Boss wants to talk to you."
-      class="net.sourceforge.pmd.lang.rule.xpath.XPathRule">
+      class="net.sourceforge.pmd.lang.rule.XPathRule">
```
Comment (adangel) [inline docs/pages/pmd/userdocs/extending/your_first_rule.md:130]: Actually, it is correct like it is: The class `XPathRule` moved with PMD 7 into the sub-package `xpath`.
So, what is currently documented on https://docs.pmd-code.org/latest/pmd_userdocs_extending_your_first_rule.html is correct for PMD 7.

(in PMD 6, it was without the sub-package `xpath`, e.g. https://docs.pmd-code.org/pmd-doc-6.55.0/pmd_userdocs_extending_your_first_rule.html).

**Comment by adangel** [comment]
No need to update the documentation, it is already correct.

**Pull request closed without merging**